core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/scheduling/ScheduleManagerImpl.java

```diff
@@ -55,6 +57,20 @@ private ScheduleManagerImpl(final Builder builder)
                 TimeUnit.MILLISECONDS);
     }
 
+    @Override
+    public String getCurrentJobStatus()
+    {
+        ScheduledJob job = currentExecutingJob.get();
+        if (job != null)
+        {
+            String jobId = job.getId().toString();
+            return "Running Job - ID: " + jobId + ", Status: " + "Running";
```

Comment:
Why the "Status: " + "Running" construct?

Reply:
Done